src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -1968,7 +1968,7 @@ DeleteShardRow(uint64 shardId)
 	Relation pgDistShard = table_open(DistShardRelationId(), RowExclusiveLock);
 
 	ScanKeyInit(&scanKey[0], Anum_pg_dist_shard_shardid,
-				BTEqualStrategyNumber, F_INT8EQ, Int64GetDatum(shardId));
+				BTEqualStrategyNumber, F_INT8EQ, UInt64GetDatum(shardId));
```

Comment:
there is no "ubigint" in Postgres. so, in the catalog, it is bigint.

I think I prefer to keep as Int64GetDatum to make sure we use the same as the catalog.